### PR TITLE
[Doc] Spanish translation of pad_truncation.md

### PR DESCRIPTION
**Pull request by aaronjimv**
# What does this PR do?

Add the Spanish version of `pad_truncation.md` to `transformers/docs/source/es`

Fix a typo in the table of `en/pad_truncation.md`

Fixes #15947 

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

@omarespejel @sgugger @osanseviero @stevhliu

### CODE REVIEW

**Review by stevhliu — APPROVED** [review]
LGTM, thanks for also fixing the `en` version! Pinging @osanseviero for a quick review of the content :)

**Comment by aaronjimv** [comment]
Hi @stevhliu, thanks!

File: docs/source/es/pad_truncation.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  - `'only_second'`: trunca hasta una longitud máxima especificada por el argumento `max_length` o la longitud máxima
+    aceptada por el modelo si no se proporciona `max_length` (`max_length=None`). Esto solo truncará
+    la segunda oración de un par si se proporciona un par de secuencias (o un lote de pares de secuencias).
+  - `'only_first'`: trunca hasta una longitud máxima especificada por el argumento `max_length` o la longitud máxima
+    aceptada por el modelo si no se proporciona `max_length` (`max_length=None`). Esto solo truncará
+    la primera oración de un par si se proporciona un par de secuencias (o un lote de pares de secuencias).
+  - `False` o `'do_not_truncate'`: no se aplica truncamiento. Este es el comportamiento predeterminado.
+
+El argumento `max_length` controla la longitud del Padding y el Truncation. Puede ser un número entero o `None`, en cuyo caso se establecerá automáticamente en la longitud máxima que el modelo puede aceptar. Si el modelo no tiene una longitud máxima de entrada específica, se desactiva el Truncation o el Padding hasta `max_length`.
```
Comment (osanseviero) [inline docs/source/es/pad_truncation.md:43]: I would be consistent with calling it truncamiento o Truncation, same for padding.

**Review by osanseviero — APPROVED** [review]
Nice! I did a quick review and overall it looks good!

File: docs/source/es/pad_truncation.md (hunk 1 above)
Comment (aaronjimv) [reply docs/source/es/pad_truncation.md]: Hi @osanseviero, will be ok in this way?

> El argumento `max_length` controla la longitud del Padding (relleno) y el Truncation (truncamiento). Puede ser un número entero o `None`, en cuyo caso se establecerá automáticamente en la longitud máxima que el modelo puede aceptar. Si el modelo no tiene una longitud máxima de entrada específica, se desactiva el Truncation o el Padding hasta `max_length`.

Comment (osanseviero) [reply docs/source/es/pad_truncation.md]: Hi there! My feedback was that in some sections you use the word in English and others in Spanish. E.g. 

* line 23: "relleno"
* 43: "Padding"
* 45: padding
* 19: padding
* 25: "relleno"

My suggestion would be to only say `El relleno (tambien conocido como "Padding") as you already do in line 19, and afterwards only say padding or relleno, but not both. Imo you can use "relleno" and just said padding when you're explicitly saying set the `padding` parameter or similar

Comment (aaronjimv) [reply docs/source/es/pad_truncation.md]: Ok, I get it. Thanks.

**Comment by aaronjimv** [comment]
Hi @osanseviero, thanks for the help. Please let me know anything else.

**Pull request merged**